astronomer/providers/amazon/aws/triggers/emr.py

```diff
@@ -0,0 +1,76 @@
+import asyncio
+from typing import Any, AsyncIterator, Dict, Iterable, Optional, Tuple
+
+from airflow.triggers.base import BaseTrigger, TriggerEvent
+
+from astronomer.providers.amazon.aws.hooks.emr import EmrStepSensorHookAsync
+
+
+class EmrStepSensorTrigger(BaseTrigger):
+    """
+    A trigger that fires once AWS EMR cluster step reach either target or failed state
+
+    :param job_flow_id: job_flow_id which contains the step check the state of
+    :param step_id: step to check the state of
+    :param aws_conn_id: aws connection to use, defaults to 'aws_default'
+    :param poke_interval: Time in seconds to wait between two consecutive call to
+        check emr cluster step state
+    :target_states: the target states, sensor waits until
+        step reaches any of these states
+    :failed_states: the failure states, sensor fails when
+        step reaches any of these states
```

Comment:
```suggestion
    :param target_states: the target states, sensor waits until
        step reaches any of these states
    :param failed_states: the failure states, sensor fails when
        step reaches any of these states
```

Reply:
Updated the docstring e9eb74b1fe0f7b7d015eeb34cc2acde939371b7e